README.md

```diff
@@ -115,8 +115,10 @@ zstyle ':completion:*' menu no
 # preview directory's content with eza when completing cd
 zstyle ':fzf-tab:complete:cd:*' fzf-preview 'eza -1 --color=always $realpath'
 # custom fzf flags
-# NOTE: fzf-tab does not follow FZF_DEFAULT_OPTS
+# NOTE: fzf-tab does not follow FZF_DEFAULT_OPTS by default
 zstyle ':fzf-tab:*' fzf-flags --color=fg:1,fg+:2 --bind=tab:accept
+# To make fzf-tab follow FZF_DEFAULT_OPTS
```

Comment:
I think we should warn user that this may cause unexpected behavior.

Reply:
@PrayagS I think you can mention here there are other plugins that overrides the value of fzf-flags directly, creating bugs.

Now I was wondering if overriding the fzf-flags is the right thing to do for an external plugin, it should do something like: fzf-flags = fzf-flags + <extra_things>?

Reply:
@Aloxaf Updated the README. Please review [`380daee` (#479)](https://github.com/Aloxaf/fzf-tab/pull/479/commits/380daeeb8af1972b901dc873c4fe4a74f897bb21).

@pablospe My comment on Freed-Wu/fzf-tab-source causing a conflict is a hunch. Because when I was reviewing its code, it overrides it specifically for a completion type (see https://github.com/Freed-Wu/fzf-tab-source/blob/aabde06d1e82b839a350a8a1f5f5df3d069748fc/fzf-tab-source.plugin.zsh#L31). The more likely reason for it to not work could be just a parsing issue with the flags.